CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # PRQL Changelog
 
+## 0.3.1 - Unreleased
+
+- Update Github Actions and Workflows to current version numbers
+  (and avoid using Node12 anymore)
```

Comment:
Thanks! Leading by example :)

Would you be up for adding your GH handle? No pressure

Reply:
Thanks, but I'm OK standing on the sidelines.